build/makelib/golang.mk

```diff
@@ -117,7 +117,7 @@ go.init:
 	@:
 
 .PHONY: go.build
-go.build:
+go.build: go.mod.vendor
```

Comment:
Instead of adding this vendoring step here, how about adding it next to `go.test.unit` down on line 131, which is the step where we're seeing the failure in jenkins:

```
go.test.unit: go.mod.vendor $(GOJUNIT)
```

This is working for me locally